diff --git a/mmdet/__init__.py b/mmdet/__init__.py
index 3ac884ac..20633845 100644
--- a/mmdet/__init__.py
+++ b/mmdet/__init__.py
@@ -6,7 +6,7 @@ from mmengine.utils import digit_version
 from .version import __version__, version_info
 
 mmcv_minimum_version = '2.0.0rc4'
-mmcv_maximum_version = '2.2.0'
+mmcv_maximum_version = '2.3.0'
 mmcv_version = digit_version(mmcv.__version__)
 
 mmengine_minimum_version = '0.7.1'
